.github/workflows/cezzis-mcp-cicd.yaml

```
@@ -37,4 +61,28 @@
+      arm_subscription_id: ${{ vars.ARM_SUBSCRIPTION_ID }}
+      arm_tenant_id: ${{ vars.ARM_TENANT_ID }}
+      backend_resource_group_name: rg-vec-eus-administration-001
+      backend_storage_account_name: stveceusterraformstat001
+      backend_container_name: terraform-cocktails
+      backend_blob_key: cocktails/prd-cocktails-mcp.tfstate
+      command_line_args: '-var image_tag=${{ github.sha }}'
+    secrets:
+      arm_client_secret: ${{ secrets.ARM_CLIENT_SECRET }}
+      terraform_module_repo_access_key_secret: ${{ secrets.TERRAFORM_MODULE_REPO_ACCESS_KEY_SECRET }}
+      command_line_args: '-var "b2c_terraform_app_registration_client_id=${{ secrets.PRD_B2C_TERRAFORM_APP_REGISTRATION_CLIENT_ID }}" -var "b2c_terraform_app_registration_client_secret=${{ secrets.PRD_B2C_TERRAFORM_APP_REGISTRATION_CLIENT_SECRET }}"'
+    
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)